### ISSUE TITLE
.charles.yml test

### ISSUE THREAD

**Issue by amihaiemil** [amihaiemil/eva#34]
This is for testing of @charlesmike's .yml config file

**Comment by amihaiemil** [comment]
@g4s8 Can you help me a bit? I have a ``.charles.yml`` file, which declares commanders for my bot (normally only the repo's owner can command it).

I've added you as a commander, can you try a few commands?
So far, it has ``indexsite`` (e.g. ``@charlesmike index this site pls``), ``delete`` (e.g. ``@charlesmike delete `eva` index``) and a few others. 

Can you pls try a delete and then an indexsite when you have 1 min? Thanks.

**Comment by charlesmike** [comment]
> @g4s8 Can you help me a bit? I have a ``.charles.yml`` file, which declares commanders for my bot (normally only the repo's owner can command it).

I've added you as a commander, can you try a few commands?
So far, it has ``indexsite`` (e.g. ``@charlesmike index this site pls``), ``delete`` (e.g. ``@charlesmike delete `eva` index``) and a few others. 

Can you pls try a delete and then an indexsite when you have 1 min? Thanks.


@amihaiemil thank you for the command, the indexing process started just now. I will let you know of the outcome shortly. In the meantime, you can check the logs [here](http://charles.amihaiemil.com/logs.html?log=/ce3a5ad8-6058-4c8b-b858-6c826dd8863b.log).

**Comment by charlesmike** [comment]
> @g4s8 Can you help me a bit? I have a ``.charles.yml`` file, which declares commanders for my bot (normally only the repo's owner can command it).

I've added you as a commander, can you try a few commands?
So far, it has ``indexsite`` (e.g. ``@charlesmike index this site pls``), ``delete`` (e.g. ``@charlesmike delete `eva` index``) and a few others. 

Can you pls try a delete and then an indexsite when you have 1 min? Thanks.


@amihaiemil Index command executed successfully! 
 Add the widget [script](https://github.com/opencharles/charles-search-box#reactjs-search-box-component) to the website in order to make the search functionality available.

 See [logs](http://charles.amihaiemil.com/logs.html?log=/ce3a5ad8-6058-4c8b-b858-6c826dd8863b.log) for more details. 
 If you notice any inexplicable errors, please open an issue [here](https://github.com/opencharles/charles-rest/issues/new)

**Comment by charlesmike** [comment]
waiting to be tested

**Comment by charlesmike** [comment]
> @g4s8 Can you help me a bit? I have a ``.charles.yml`` file, which declares commanders for my bot (normally only the repo's owner can command it).

I've added you as a commander, can you try a few commands?
So far, it has ``indexsite`` (e.g. ``@charlesmike index this site pls``), ``delete`` (e.g. ``@charlesmike delete `eva` index``) and a few others. 

Can you pls try a delete and then an indexsite when you have 1 min? Thanks.


@amihaiemil Index command executed successfully! 
 Add the widget [script](https://github.com/opencharles/charles-search-box#reactjs-search-box-component) to the website in order to make the search functionality available.

 See [logs](http://charles.amihaiemil.com/logs.html?log=/73b19883-849a-42b0-abbf-b4caaf176344.log) for more details. 
 If you notice any inexplicable errors, please open an issue [here](https://github.com/opencharles/charles-rest/issues/new)

**Comment by g4s8** [comment]
@charlesmike delete `eva` index

**Comment by charlesmike** [comment]
> @charlesmike delete `eva` index

@g4s8 the `eva` index was successfully deleted. 
Don't forget to remove the script form your website since the search will not work anymore.
See [logs](http://charles.amihaiemil.com/logs.html?log=/62950c18-7bf5-465a-a5e5-74f9a60f0f62.log) for details.

If you can spare a minute, please consider opening an issue [here](https://github.com/opencharles/charles-rest/issues/new),let us know why you stopped using this service, what you didn't like and what we could improve. Thanks!

**Comment by g4s8** [comment]
@charlesmike index this site pls

**Comment by charlesmike** [comment]
> @charlesmike index this site pls

@g4s8 thank you for the command, the indexing process started just now. I will let you know of the outcome shortly. In the meantime, you can check the logs [here](http://charles.amihaiemil.com/logs.html?log=/ca884ca8-83b4-47f3-8de5-58a5ef195787.log).

**Comment by charlesmike** [comment]
> @charlesmike index this site pls

@g4s8 Index command executed successfully! 
 Add the widget [script](https://github.com/opencharles/charles-search-box#reactjs-search-box-component) to the website in order to make the search functionality available.

 See [logs](http://charles.amihaiemil.com/logs.html?log=/ca884ca8-83b4-47f3-8de5-58a5ef195787.log) for more details. 
 If you notice any inexplicable errors, please open an issue [here](https://github.com/opencharles/charles-rest/issues/new)

**Comment by g4s8** [comment]
@amihaiemil see, I did it. BTW do you have some examples in this project? I found blog post, but can't find any sample code

**Comment by amihaiemil** [comment]
@g4s8 seems to work fine, thanks!

You mean samples for eva?

**Comment by g4s8** [comment]
@amihaiemil yes

**Comment by amihaiemil** [comment]
@g4s8 the one in the post is a complete example for the backpack's problem, it should work. Other examples you can find in the unit tests and it also has a website: eva.amihaiemil.com

That's it :D but if you have a concrete case for it, maybe I can help you

**Comment by g4s8** [comment]
@charlesmike hello

**Comment by charlesmike** [comment]
> @charlesmike hello

Hi @g4s8! I can help you index your Github-hosted website, provided that the repository name respects the format ``owner.github.io`` (or has a ``gh-pages`` branch) and the command is given by the repo's owner.

More on commands [here](http://charles.amihaiemil.com/doc.html#commands).
Also, [this](http://www.amihaiemil.com/2017/05/23/meet-charles-michael.html) blog post might be useful.

**Comment by amihaiemil** [comment]
@g4s8 yes, everyone can say hello, in any repo. There is a checker spinning, so it should answer anywhere. But there is also the option of webhook, I'll update the docs soon :D

**Comment by g4s8** [comment]
@amihaiemil I've got interested) going to try it about next week
